core/src/test/java/hudson/UtilTest.java

```diff
@@ -558,7 +560,7 @@ public void resolveSymlinkToFile() throws Exception {
         File aa = new File(a, "aa");
         aa.mkdirs();
         File aaTxt = new File(aa, "aa.txt");
-        FileUtils.write(aaTxt, "aa");
+        FileUtils.write(aaTxt, "aa", StandardCharsets.US_ASCII, false);
```

Comment:
Why do you not use UTF_8?

Reply:
Both are used in the codebase. If someone wants to make a PR that converts `US_ASCII` to `UTF_8` and bans the former, I'm fine w/ that.

It was admittedly arbitrary, but since the content is just `a`s, it didn't seem worth asking for `UTF-8`.

If people insist, I can add a commit to change this. But, tbh, I'd rather do a PR across the codebase than an extra commit here followed by an extra PR to clean the rest of the codebase.

(If reviewers here are ok w/ me writing the commit, I'd be slightly more inclined to fix these two files here.)